Orange/widgets/model/owneuralnetwork.py

```diff
@@ -83,6 +83,7 @@ class OWNNLearner(OWBaseLearner):
     max_iterations = Setting(200)
     alpha_index = Setting(0)
     settings_version = 1
+    random_state = 1
```

Comment:
This should be `random_state = Setting(1)` so the widget will remember this setting. Also, it should be above `settings_version = 1` -- because it's a setting, while `settings_version` has a number that is increased when settings are changed in such a way that they need to be migrated.

Reply:
I agree.